### ISSUE TITLE
Preserve console functions 

### ISSUE THREAD

**Issue by princejwesley** [chinchang/screenlog.js#2]
[console](https://github.com/chinchang/screenlog.js/blob/master/screenlog.js#L48) functions should be preserved.

``` js

let clog = console.log;
..
function log() {
  ..
  clog.apply(console, arguments);
}
```

**Comment by Tatsh** [comment]
Would like to see this too.

**Comment by chinchang** [comment]
this has been fixed in #10 and is available in 0.1.3
